homeassistant/components/habitica/__init__.py

```diff
@@ -122,6 +143,73 @@ class HAHabitipyAsync(HabitipyAsync):
         def __call__(self, **kwargs):
             return super().__call__(websession, **kwargs)
 
+    async def cast_skill(call: ServiceCall) -> ServiceResponse:
+        """Skill action."""
+        entry: HabiticaConfigEntry | None
+        if entry := hass.config_entries.async_get_entry(call.data[ATTR_CONFIG_ENTRY]):
+            coordinator = entry.runtime_data
+            skill = {
+                "pickpocket": {"spellId": "pickPocket", "cost": "10 MP"},
+                "backstab": {"spellId": "backStab", "cost": "15 MP"},
+                "smash": {"spellId": "smash", "cost": "10 MP"},
+                "fireball": {"spellId": "fireball", "cost": "10 MP"},
+            }
+            try:
+                task_id = next(
+                    task["id"]
+                    for task in coordinator.data.tasks
+                    if call.data[ATTR_TASK] in (task["id"], task.get("alias"))
+                    or call.data[ATTR_TASK] == task["text"]
+                )
+            except StopIteration as e:
+                raise ServiceValidationError(
+                    translation_domain=DOMAIN,
+                    translation_key="service_call_task_not_found",
+                    translation_placeholders={"task": f"'{call.data[ATTR_TASK]}'"},
```

Comment:
Why do we pre-wrap them in `'`

Reply:
Wanted to highlight it as `code` but backticks are not supported by the toast notifications in the UI. Or should I use backticks nonetheless?  Or nothing at all?